### ISSUE TITLE
A symbolic link has to be created when installing the library

### ISSUE THREAD

**Issue by PerfectCarl** [inizan-yannick/afrodite#3]
I don't know automake enough to do it enough but this command needs to be run so application can find afrodite: 

```
sudo ln -s  /usr/lib/libafrodite-0.28.so /usr/lib/libafrodite.so

```

**Comment by inizan-yannick** [comment]
no, version is the vala version, so you can't build an application with another version of vala

**Comment by PerfectCarl** [comment]
Well the ValaBinding addin [expects](https://github.com/PerfectCarl/ValaBinding/blob/master/Parser/Afrodite.cs#L90) to load `libafrodite.so` so I tried to get what it wanted.

I am not sure how the addin is supposed to support different version of vala: especially now that versions differ not a lot - most vala release are vapi updates.
Different releases of the addin for different versions of vala/afrodite sounds like a packaging nightmare.

What do you reckon?
